Replace grunt service initialisation with bash script

hfaulds opened the pull request:
At the moment the `grunt install` task clones all services and runs `npm install` but it doesn't respect the node version of the given service. Some of the binaries can be different between node versions so to get everything working I had to remove `node_modules` files in each service and run `nvm install && npm install`.

Nvm isn't accessible through the processes spawned by grunt because it's reliant on special configuration in the users shell config.

This PR does two things:
1. remove the parts of the grunt task which run `npm install`
1. add a shell script which can be sourced, which runs `nvm install && npm install` for each service. I went with `nvm install` as this is a one off configuration task

I'm unsure on what documentation if any to update

henryoswald commented:
Thanks, how would you use this script as part of the grunt install process?

hfaulds commented:
@henryoswald sorry I didn't explain this very well.

As it stands with this PR one would have to run `grunt install` and then separately `source bin/install-services` to get started.

I couldn't work out a way of changing node version within the non login-shell shell spawned by grunt. I'm not particularly happy with having two steps to get set up but it seems better than the current grunt task which creates a broken installation.

henryoswald commented:
sure, the current grunt task isn't really used by people. How about adding a log line at the end of the grunt job telling the user to run install-services?

hfaulds commented:
@henryoswald I've added a postinstall message and a bit more logging to the install-services script.

I removed all of my services folders, ran `grunt install` then `source bin/install-services` and checked basic editing / preview functionality. Everything worked as expected.

iselcuk commented:
@hfaulds I had a fully working docker container but I wanted to change git repositories with forked ones to be able to change it later. Removed all of my service folders and followed the same steps but it resulted like this: 
![screenshot from 2017-11-13 11 01 43](https://user-images.githubusercontent.com/11681283/32715220-67230cf0-c862-11e7-9caa-6b646fa84a23.png)

- Same steps without changing git repo urls and it resulted same. 
- Same steps with backing up web service with another name like "weboriginal" and installing web via `grunt install` and executing `source bin/install-services` after that resulted same.
- Removing "web" and changing "weboriginal" to "web" works but it is where we started...

How can I edit some of the services like web-sharelatex over my git repo?
